app/blueprints/search.py

```diff
@@ -1,11 +1,54 @@
+import re
+from functools import wraps
+
 from flask import Blueprint, current_app, jsonify, request
-from index_search import search as azure_index_search
+from index_search import AzureIndexSearchQueryError, search
+
+from app.finesse_data import FinesseDataFetchException, fetch_data
+
+search_blueprint = Blueprint("finesse", __name__)
+
+
+def require_non_empty_query(f):
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        query = request.json.get("query")
+        if not query:
+            return jsonify({"message": current_app.config["ERROR_EMPTY_QUERY"]}), 400
+        return f(*args, **kwargs)
 
-search = Blueprint("finesse", __name__)
+    return decorated_function
+
+
+def sanitize(input):
+    return re.sub("[^\w\s\d\"#\$%&'\(\)\*\+,-\./:;<=>\?@\[\\\]\^_`{\|}~]+", "", input)
```

Comment:
make this configurable with regexp value as default

Reply:
Done